reconstructSPI/iterative_refinement/expectation_maximization.py

```diff
@@ -576,17 +647,22 @@ def apply_wiener_filter(projection, ctf, small_number):
         return projection_wfilter_f
 
     @staticmethod
-    def insert_slice(slice_real, xyz, n_pix):
+    def insert_slice(slice_real, xy_rotated, xyz):
```

Comment:
Doesn't it work out if you just pass in one n**2 rotated plane of coordinates, with no z offset. 

I'd like to hear an argument or evidence for this: `(i.e. a rotated 2D plane with some small added z-depth to give "volume" to the slice in order for interpolation to be feasible)`